mediainfolib.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: mediainfolib
+  version: 25.03
+  epoch: 0
+  description: "Utility for displaying technical and tag data for media files"
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates
+      - git
+      - build-base
+      - libtool
+      - pkgconf
+      - pkgconf-dev
```

Comment:
I'm not sure why you'd need `pkgconf-dev` - isn't that just for building tools against `libpkgconf`?